src/projetos/CadastroProjetosPage.tsx

```
@@ -646,6 +646,8 @@ export function CadastroProjetoPage() {
                               <UserTableDialog
                                 selecionados={form.getValues('equipeExecutora')}
                                 onChange={field.onChange}
+                                projetoId={projetoSelecionado?.id || Number(id) || 0}
```

ESLint

[error] Replace `projetoSelecionado?.id·||·Number(id)·||·0}` with `⏎··································projetoSelecionado?.id·||·Number(id)·||·0`

(prettier/prettier)